src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -1056,6 +1056,11 @@ public override BoundNode VisitInterpolatedString(BoundInterpolatedString node)
             return null;
         }
 
+        public override BoundNode VisitInterpolatedString(BoundInterpolatedString node)
```

Comment:
Are we not expecting to run into an uconverted node? I think we should still override the visit method and perhaps throw.